### PR TITLE
Mi información básica

### PR DESCRIPTION
**Pull request by Cesar1520**
Hello dear colleagues please review my work

### CODE REVIEW

File: ejercicio.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Sub recolecta()
+    donaron = 0
+    noDonaron = 0
+    totalRecaudo = 0
+    promedioRecaudo = 0
+    diezMil = 0
+    siguiente = "si"
+
+    While siguiente = "si" And totalRecaudo < 3000000
```
Comment (EmanuelObredor18) [inline ejercicio.md]: What is the function of the variable "siguiente"?

File: ejercicio.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        donar = Int(InputBox("Ingrese la cantidad a donar"))
+        totalRecaudo = donar + totalRecaudo
+        If donar >= 10000 Then
+            diezMil = diezMil + 1
+        End If
+        donaron = donaron + 1
+        siguiente = InputBox("Alguien más va a donar?")
```
Comment (EmanuelObredor18) [inline ejercicio.md]: There could be conflicts here when executing this new InputBox. You already have one of these assigning above.

**Review by EmanuelObredor18 — CHANGES_REQUESTED** [review]
You have too many bugs to fix. Please check your flowchart, it's a bit messy. The rest of the recommendations can be found in the comments.

**Review by Luisafernanda29 — APPROVED** [review]
Good job!

**Review by EmanuelObredor18 — APPROVED** [review]
Well done @Cesar1520 .

**Review by GoJhon — APPROVED** [review]
@Cesar1520 Good job! keep it up!

**Review by hguzman — APPROVED** [review]
Ok

**Pull request merged**